### DISCUSSION TITLE
Orientations Question

### DISCUSSION THREAD

**Discussion by JoshTheMenace** [gempy-project/gempy#619] [Q&A]
**What is your question?**
I am pretty new to geological modeling, and I am loving gempy so far. I am just not sure what orientations really do and how a geologist gather that data in the real world. Maybe I am not understanding the term very well. Is there a guide for the terminology that gempy uses or a guide to understanding orientations better? I am not seeing much information on orientations in the documentation and tutorials.

**Comment by AlexanderJuestel** [comment]
Dear @TheArcher1958, 

thanks for trying out GemPy! Orientations are fairly simple to understand. You can measure an orientation, consisting of an dip (angle) and azimuth (angle) value, in the field with a geological compass. You basically measure how a geological layer is oriented in space. 

In GemPy you can just use these measured orientations to define the gradient of the scalar field that is calculated to obtain the location of your modelled surface in the end. A dataframe or a csv file containing these orientations are one of the two main input types for GemPy.

Let us know if you have any further questions!
